src/rest_vol.c

```diff
@@ -4125,8 +4124,7 @@ RV_curl_delete(CURL *curl_handle, server_info_t *server_info, const char *reques
 
     strcpy(host_header, host_string);
 
-    curl_headers_local = curl_slist_append(
-        curl_headers_local, strncat(host_header, filename, host_header_len - strlen(host_string) - 1));
```

Comment:
Why were these converted back to `strcat` calls?

Reply:
The lengths provided to `strncat` were pulled from the length of the target strings anyway, so having these be `strncat` didn't really accomplish anything.

Reply:
Hmm, in this case yes they weren't doing much, though they're more future-proof than a regular `strcat` in case someone adds more things to the header in the future and forgets to allocate the space correctly. Really, I think most of these should just be switched to a single `snprintf` anyway to avoid most of the problems of having to manage the buffer size.